__docs__/wonder-blocks-modal/flexible-dialog.stories.tsx

```diff
@@ -251,6 +299,114 @@ export const WithStyle: StoryComponentType = {
     ),
 };
 
+/**
+ *
+ * A FlexibleDialog will adjust with long contents, instead of fixing its height.
+ */
+export const WithLongContents: StoryComponentType = {
+    render: () => (
+        <View style={styles.previewSizer}>
+            <View style={styles.modalPositioner}>
+                <FlexibleDialog
+                    title={<Heading>Hello, world!</Heading>}
+                    content={
+                        <>
+                            <BodyText>
+                                {`Lorem ipsum dolor sit amet, consectetur adipiscing
```

Comment:
Tip: We also have lorem ipsum constants for common text to test with! https://github.com/Khan/wonder-blocks/blob/main/__docs__/components/text-for-testing.ts